README.md

```diff
@@ -16,16 +16,25 @@ The FastAPI application provides a simple endpoint at the root (`/`) that return
 
 1. Open the project in VSCode.
 2. When prompted, open the project inside the container. (Or press `F1`, type `Remote-Containers: Reopen in Container`).
-3. Once inside the container, VSCode will automatically bind the project directory and forward port `5001` for the FastAPI application.
-4. Visit [Localhost:5001](http://localhost:5001/) in your browser to see the current Unix timestamp returned by the FastAPI app.
-5. Access the API Swagger docs [here](http://localhost:5001/docs).
+3. Once inside the container, open VSCode Terminal and run these commands:
+```
+cd /workspaces/fastapi-example/src
+uvicorn main:app --port <PORT>
+```
+This will start the FastAPI application, and you can access it on port <PORT>
+
+4. Visit `http://localhost:<PORT>`. in your browser to see the current Unix timestamp returned by the FastAPI app.
+5. Access the API Swagger docs at `http://localhost:<PORT>/docs`.
+
 
-### Manually Running the App (If Needed)
+### Customizing ASGI Server
 
-While inside the VSCode terminal connected to the container:
+The project currently uses uvicorn as its ASGI server, as specified in the `requirements.txt` file. If you wish to use another ASGI server like `hypercorn` or `daphne`, simply update the `requirements.txt` file and modify the startup command accordingly in the `devcontainer.json` file and the `Dockerfile`. Make sure the server you choose is compatible with FastAPI.
+
+### Running Tests
+
+After setting up the project, you can run the unit tests to ensure everything works as expected:
 
 ```bash
-cd /workspaces/fastapi_example/src
```

Comment:
don't use absolute paths here, they only make sense within a devcontainer.

you can assume the user is in the root of the project, so :

```
cd src
```